app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -1,5 +1,5 @@
 <% document_contents_guidance = capture do %>
-  <%= I18n.exists?("document_types.#{edition.document_type.id}.fields.#{field.id}.guidance") ? render_govspeak(t("document_types.#{edition.document_type.id}.fields.#{field.id}.guidance.body_govspeak")) : nil %>
+  <%= t_doctype_key_exists?("document_types.#{edition.document_type.id}.fields.#{field.id}.guidance") ? render_govspeak(t_doctype("document_types.#{edition.document_type.id}.fields.#{field.id}.guidance.body_govspeak")) : nil %>
```

Comment:
Since guidance is optional, perhaps we could use a dedicated `t_doctype_guidance` helper that's allowed to return nil - this would DRY-up the ternary, as well as most of the elements of the locale key.

Reply:
Great idea 👍 I wasn't sure what to call it but this sounds nice.

Reply:
This idea is superseded by using raw I18n instead (we don't anticipate setting any defaults).